### PR TITLE
Add nodejs-graphql package

### PR DESCRIPTION
**Pull request by xprazak2**

### CODE REVIEW

**Comment by xprazak2** [comment]
Needed for https://github.com/theforeman/foreman-packaging/pull/6692 to build

File: comps/comps-foreman-plugins-el8.xml [hunk 1]
```diff
@@ -183,6 +183,7 @@
         Do not edit this section manually and use ./comps_doc.sh script
       -->
       <!--RGD-START-->
+      <packagereq type="default">rubygem-acts_as_list-doc</packagereq>
```
Comment (evgeni) [inline comps/comps-foreman-plugins-el8.xml:186]: this seems unrelated?

File: comps/comps-foreman-rhel7.xml [hunk 2]
```diff
@@ -33,6 +33,7 @@
       <packagereq type="default">foreman-service</packagereq>
       <packagereq type="default">foreman-telemetry</packagereq>
       <packagereq type="default">foreman-vmware</packagereq>
+      <packagereq type="default">nodejs-graphql</packagereq>
```
Comment (evgeni) [inline comps/comps-foreman-rhel7.xml:36]: I'd expect this to be tfm-nodejs-graphql for EL7?

(and the next line too, but that's not part of this pr)

**Review by evgeni — APPROVED** [review]

**Comment by evgeni** [comment]
```console
% koji add-pkg foreman-nightly-rhel7 tfm-nodejs-graphql --owner=kojiadm
Adding 1 packages to tag foreman-nightly-rhel7
% koji add-pkg foreman-nightly-el8 nodejs-graphql --owner=kojiadm 
Adding 1 packages to tag foreman-nightly-el8
```
